proof-systems/src/darlin/tests/mod.rs

```diff
@@ -160,21 +160,19 @@ mod test {
             pcds[idx] = fake_pcds.unwrap()[idx].clone();
             vks[idx] = fake_vks.unwrap()[idx].clone();
 
-            let result = verify_aggregated_proofs::<G1, G2, D, R>(
+            let result = batch_verify_proofs::<G1, G2, D, R>(
```

Comment:
Why ? We should have a test also for accumulation and verification of accumulation proofs

Reply:
Oops, copypaste ( fixed